### DISCUSSION TITLE
World Package 1.4.1 업데이트하면 ZepetoWorldMultiplay사라집니다.

### DISCUSSION THREAD

**Discussion by mzdokebi** [naverz/zepeto-studio-kor#874] [5. Q&A]
유니티 에디터 Add Component에서도 검색이 안됩니다.
이전에도....
가이드의 예제를 보면 ClientStarter를 만들어서 Multiplay를 인자로 넘기는 부분이 있는데 이게 실행시 인자로 넘긴 Multiplay(ZepetoWorldMultiplay)가 사라지고 캐릭터 생성 등 멀티플레이 관련 기능이 멈추는 현상이 있어서 Multiplay 오브젝트를 Update에서 계속 검색하고 검색이 되면 ZepetoWorldMultiplay 컴포넌트를 받아서 사용하고 있었습니다.
기존에는 이런식으로 ZepetoWorldMultiplay가 실행시 사라지는 문제를 해결하고 있었지만
이번 업데이트를 받으면 프로젝트에서 ZepetoWorldMultiplay가 그냥 사라집니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요. 해당 이슈는 1.5.0에서 해결될 예정이며 
ASSETS > Reimport All 을 하신 후 다시 찾아보시기를 추천 드립니다.
https://github.com/naverz/zepeto-studio-kor/discussions/797#discussioncomment-2816174

감사합니다.
